File: projects/python/lotf-game/scripts/sprites.py
import pygame

playerSprites = {
    'left': {
        'idle': pygame.transform.scale(pygame.image.load('../assets/sprites/lefttIdle.png'), (36, 96)),
        'run': [
            pygame.transform.scale(pygame.image.load('../assets/sprites/leftRun0.png'), (36, 96)),
            pygame.transform.scale(pygame.image.load('../assets/sprites/rightRun0.png'), (36, 96))
        ]
    },
    'right': {
        'idle': pygame.transform.scale(pygame.image.load('../assets/sprites/rightIdle.png'), (36, 96)),
        'run': [
            pygame.transform.scale(pygame.image.load('../assets/sprites/rightRun0.png'), (36, 96)),
            pygame.transform.scale(pygame.image.load('../assets/sprites/rightRun1.png'), (36, 96))
        ]
    }
}

nythanSprites = {}

philSprites = {}

erikSprites = {}

savasSprites = {}

File: projects/game-engine/examples/multiplayer/client/JS/Sprites.js
const Sprites = {
    LocalPlayer: {
        idle: [ new Image() ],
        move: [ new Image(), new Image(), new Image(), new Image() ]
    },

    NonLocalPlayer: {
        idle: [ new Image() ],
        move: [ new Image(), new Image(), new Image(), new Image() ]
    }
}

for (let animation in Sprites.LocalPlayer) {
    for (let i = 0; i < Sprites.LocalPlayer[animation].length; ++i) {
        Sprites.LocalPlayer[animation][i].src = "Assets/local_player/" + animation + "_" + i + ".png";
    }
}

for (let animation in Sprites.NonLocalPlayer) {
    for (let i = 0; i < Sprites.NonLocalPlayer[animation].length; ++i) {
        Sprites.NonLocalPlayer[animation][i].src = "Assets/non_local_player/" + animation + "_" + i + ".png";
    }
}